molecule/molecule-testing/src/main/java/app/cash/molecule/testing/moleculeTesting.kt

```diff
@@ -33,20 +33,19 @@ import kotlinx.coroutines.withTimeout
 import kotlinx.coroutines.yield
 import kotlin.coroutines.CoroutineContext
 import kotlin.time.Duration
-import kotlin.time.ExperimentalTime
+import kotlin.time.Duration.Companion.milliseconds
 
 @ExperimentalCoroutinesApi
-@ExperimentalTime
 fun <T> testMolecule(
```

Comment:
Should this `Long`-accepting overload be deprecated? I don't have any good arguments to keep it long-term.

Reply:
A thumb is not an argument

Reply:
We should probably just replace this whole module with what we're using now internally...